pipeline/astro_cal.py

```diff
@@ -151,6 +152,15 @@ def __init__(self, **kwargs):
 
     # ----------------------------------------------------------------------
 
+    # TODO: should we move all of these catalog fetches into a separate file?
+    #  probably need to when doing the photometric calibration
```

Comment:
Yeah, I agree we should do this (commented on that in the photometric PR).

Reply:
...however, for DECam photometric calibration specifically, we won't actually use the Bandpass matching.  The transformations we have for all DECam filters are to the gaia Mag_G bandpass, parameterized by BP-RP, so the maximum filter overlap isn't really an issue there.  Specifically for Gaia, this probably makes sense, since Mag_G is so broad.  If we do something like a PanStarrs transformation, we may want to use the bandpass overlapping.